apps/system/utils/utils_cpuload.c

```diff
@@ -136,14 +161,35 @@ static void cpuload_print_pid_value(char *buf, void *arg)
 		printf(" %3d(%4.1f) |", cpuload_pidhash[pid_hash].count, load_ratio);
 	} else {
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-		printf(" %5s | %5s | %5s |", stat_info[PROC_STAT_CPULOAD_SHORT], stat_info[PROC_STAT_CPULOAD_MID], stat_info[PROC_STAT_CPULOAD_LONG]);
+#ifdef CONFIG_SMP
```

Comment:
There is no need to maintain a separate code for SMP case as NCPU will always be set to 1 if smp is not present.